src/universal/modules/invitation/containers/Invitation/Invitation.js

```diff
@@ -0,0 +1,154 @@
+import React, {Component, PropTypes} from 'react';
+import {push} from 'react-router-redux';
+import {connect} from 'react-redux';
+import {cashay} from 'cashay';
+import Auth0ShowLock from 'universal/components/Auth0ShowLock/Auth0ShowLock';
+import LoadingView from 'universal/components/LoadingView/LoadingView';
+import {getAuthQueryString, authedOptions} from 'universal/redux/getAuthedUser';
+import {setWelcomeActivity} from 'universal/modules/userDashboard/ducks/settingsDuck';
+import {
+  error as showError,
```

Comment:
Let's rename these in the duck itself & refactor it for all the components. Options: `showError`, `notificationError`, `errorToast`, `toastError`....